.travis.yml

```diff
@@ -1,9 +1,35 @@
 language: python
-python:
-  - 2.7
-install:
-    - pip install .
+matrix:
+  include:
+    - python: "2.7"
+    # # No python 3 (yet)
+    # - python: "3.5"
+    # # No pypy (yet, psycopg2 is the problem)
+    # - python: "pypy"
+    #   env: PYPY_VERSION="5.3.1" NO_COVERAGE=1
+cache:
+  - pip
+  - directories:
+    - $HOME/.pyenv_cache
 services:
-    - redis-server
+  - postgresql
+
+before_install:
+  - psql -c 'create database xylem;' -U postgres
+  - |
+      if [[ -n "$PYPY_VERSION" ]]; then
+        wget https://github.com/praekeltfoundation/travis-pypy/releases/download/0.1.0/setup-pypy.sh
```

Comment:
I did a travis-pyenv release finally yesterday. Check the guide: https://github.com/praekeltfoundation/travis-pyenv#example-travisyml